soc/arm/xilinx_zynqmp/Kconfig.soc

```diff
@@ -10,3 +10,11 @@ config SOC_XILINX_ZYNQMP_RPU
 	select MULTI_LEVEL_INTERRUPTS
 	select 2ND_LEVEL_INTERRUPTS
 	select PLATFORM_SPECIFIC_INIT
+
+config SOC_XILINX_ZYNQMP_APU
+	bool "Xilinx ZynqMP APU"
+	select CPU_CORTEX_A53
+	select SOC_XILINX_ZYNQMP
+	select GIC_V1
```

Comment:
For MPSoC, the Cortex-A53 cluster is connected to GIC_V2 and the Cortex-R5 cluster is connected to GIC_V1. So this should probably be corrected to "GIC_V2".

Reply:
Yes, this needs to be changed to `GIC_V2`.